ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_loss.py

```diff
@@ -468,6 +468,87 @@ def test_paddle_nll_loss(
     )
 
 
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.sigmoid_focal_loss",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=1,
+        shared_dtype=False,
+        min_num_dims=3,
+        min_dim_size=3,
+        max_num_dims=3,
+        max_dim_size=3,
+    ),
```

Comment:
This doesn't cover the entire input space

Reply:
@MuhammedAshraf2020  i had to fix the dimensions of the input to match with those of the labels as their dtype is different.